metadata_mapper/mappers/mapper.py

```diff
@@ -99,15 +99,15 @@ def to_UCLDC(self) -> dict[str, Any]:
 
     def UCLDC_map(self) -> dict:
         """
-        Defines mappings from source metdata to UCDLC that are specific
+        Defines mappings from source metadata to UCDLC that are specific
         to this implementation.
 
         All dicts returned by this method up the ancestor chain
         are merged together to produce a final result.
         """
         return {
-            "isShownAt": self.map_is_shown_at(),
-            "isShownBy": self.map_is_shown_by()
+            "is_shown_at": self.map_is_shown_at(),
+            "is_shown_by": self.map_is_shown_by()
```

Comment:
I think that these should be camelCased, based on the list of field keys we saw in today's meeting

Reply:
I checked [here](https://github.com/ucldc/rikolti/blob/main/metadata_mapper/mappers/islandora_mapper.py) because I thought the same thing, but it's snake case there. So I'm not sure.